db/migrate/20210209141131_add_commencement_date_to_trainees.rb

```diff
@@ -2,6 +2,6 @@
 
 class AddCommencementDateToTrainees < ActiveRecord::Migration[6.1]
   def change
-    add_column :trainees, :commencement_date, :date
+    add_column :trainees, :trainee_start_date, :date
```

Comment:
We shouldn't change old migration files.

Reply:
Ive unchanged it